src/applications/vaos/tests/utils/appointment.unit.spec.jsx

```diff
@@ -67,134 +279,746 @@ describe('VAOS appointment helpers', () => {
+    });
+
+    it('should return "VA Appointment" header for VA appointment request', () => {
+      expect(
+        getAppointmentTypeHeader({
+          ...vaAppointmentRequest,
+        }),
+      ).to.equal('VA Appointment');
+    });
+
+    it('should return "VA Video Connect" header for VA appointment request', () => {
+      expect(
+        getAppointmentTypeHeader({
+          ...vaAppointmentRequest,
+          visitType: 'Video Conference',
+        }),
+      ).to.equal('VA Video Connect');
+    });
+
+    it('should return "VA Appointment" header for VA appointments', () => {
+      expect(
+        getAppointmentTypeHeader({
+          clinicId: ' ',
+        }),
+      ).to.equal('VA Appointment');
+    });
+
+    it('should return "VA Video Connect" header for VA appointments', () => {
+      expect(
+        getAppointmentTypeHeader({
+          ...vaAppointment,
+        }),
+      ).to.equal('VA Video Connect');
+    });
+
+    it('should return default appointment type header', () => {
+      expect(
+        getAppointmentTypeHeader({ purposeOfVisit: 'Purpose of visit' }),
+      ).to.equal('Purpose of visit');
+    });
+  });
+
+  describe('getAppointmentDuration', () => {
+    it('should return the default appointment duration', () => {
+      expect(getAppointmentDuration({})).to.equal(60);
+    });
+    it('should return the appointment duration for VA appointment', () => {
+      expect(
+        getAppointmentDuration({
+          ...vaAppointment,
+          vdsAppointments: [{ appointmentLength: 30 }],
+        }),
+      ).to.equal(30);
+    });
+  });
+
+  describe('getAppointmentAddress', () => {
+    it('should return address for video appointment', () => {
+      const appt = {
+        vvsAppointments: [
+          {
+            dateTime: now,
+            appointmentKind: 'MOBILE_GFE',
+          },
+        ],
+      };
+      expect(getAppointmentAddress(appt)).to.equal('Video conference');
+    });
+
+    it('should return address for community care appointment', () => {
+      const appt = {
+        ...communityCareAppointment,
+        address: {
+          street: 'Street',
+          city: 'City',
+          state: 'State',
+          zipCode: 'Zipcode',
+        },
+      };
+      expect(getAppointmentAddress(appt)).to.equal(
+        'Street City, State Zipcode',
+      );
+    });
+
+    it('should return address for facility if defined', () => {
+      const facility = {
+        address: {
+          physical: {
+            address1: 'Address 1',
+            city: 'City',
+            state: 'State',
+            zip: 'Zip',
+          },
+        },
+      };
+      expect(getAppointmentAddress(vaAppointmentRequest, facility)).to.equal(
+        'Address 1 City, State Zip',
+      );
+    });
+
+    it('should return undefined for everything else', () => {
+      expect(getAppointmentAddress(vaAppointmentRequest, null)).to.be.undefined;
+    });
+  });
+
+  describe('generateICS', () => {
```

Comment:
You'll probably have to move this and the other new tests back to your other ticket.

Reply:
I think I did already.